manager/director/apps/sites/forms.py

```diff
@@ -65,6 +67,16 @@ def clean(self) -> Dict[str, Any]:
         ):
             self.add_error("name", "Project site names cannot start with a number")
 
+        if len(cleaned_data["users"]) == 0:
```

Comment:
Since you have a `"users" in cleaned_data.keys()` check below, for consistency I would do this:
```suggestion
        if len(cleaned_data.get("users", [])) == 0:
```